client/styles.css

```diff
@@ -483,6 +483,9 @@ INTRO
     background: #46426f;
   }
 }
+.userName {
+  color: #46426f
```

Comment:
I'm wondering if we can un-bold the 'Hi, {username}' and add a bit more spacing between it and the profile button. Looks a bit cramped at the moment. cc: @dshelters